### PR TITLE
New post about GSoC

### PR DESCRIPTION
**Pull request by crisclacerda**

### CODE REVIEW

File: pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown:"extra" %}
+
+This year Mixxx is again taking part on Google Summer of Code, and have 3 students working full-time during 3 months to make Mixxx even more awesome.
+After one month of community bond period, last week they started coding. Cristiano and Harshit, that are working towards the same end feature, a new beatgrid format capable of handling not only beats, but also downbeats, phrases and sections as well varying tempos and time signatures have already opened a Pull Request each on our main repository. [#2844](https://github.com/mixxxdj/mixxx/pull/2844), [#2847](https://github.com/mixxxdj/mixxx/pull/)
```
Comment (ronso0) [inline pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html]:
```suggestion
After one month of community bond period, last week they started coding. Cristiano and Harshit, who are both working towards the same final feature, a new beatgrid format capable of handling not only beats, but also downbeats, phrases and sections as well varying tempos and time signatures have already opened a Pull Request each on our main repository. [#2844](https://github.com/mixxxdj/mixxx/pull/2844), [#2847](https://github.com/mixxxdj/mixxx/pull/)
```

File: pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Cristiano which is focosing his work on the auto detection of such features is making weekly vlogs on youtube to share his progress with the community. The first one is available here:
```
Comment (ronso0) [inline pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html]:
```suggestion
Cristiano which is focusing his work on the auto detection of such features is making weekly vlogs on youtube to share his progress with the community. The first one is available here:
```

**Comment by Be-ing** [comment]
add to whitelist

File: pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/rVJ7pBWHp8E" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Janek, which is working on a feature similar to serato flip, had the great idea and prepered a community call, where the new members and (old members alike) had the opportunity to get to know each other better and put some faces behind their usernames.
```
Comment (ronso0) [inline pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html]:
```suggestion
Janek, who is working on a feature similar to serato flip, had the great idea and prepered a community call, where the new old members alike had the opportunity to get to know each other better and put some faces to their usernames. 
```

File: pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+<iframe width="560" height="315" src="https://www.youtube.com/embed/rVJ7pBWHp8E" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Janek, which is working on a feature similar to serato flip, had the great idea and prepered a community call, where the new members and (old members alike) had the opportunity to get to know each other better and put some faces behind their usernames. 
+
+We are looking forward for a new call to discuss our GSoC projects with our students, mentors and the rest of the community.
+Stay tuned on [Zulip](https://mixxx.zulipchat.com/)
+
+
+{% endfilter %}
+{% endblock %}
```
Comment (ronso0) [inline pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html]:
```suggestion
{% endblock %}

```

**Review by ronso0 — CHANGES_REQUESTED** [review]

File: pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+{% load markup %}
+{% filter markdown:"extra" %}
+
+This year Mixxx is again taking part on Google Summer of Code, and have 3 students working full-time during 3 months to make Mixxx even more awesome.
+After one month of community bond period, last week they started coding. Cristiano and Harshit, who are both working towards the same final feature, a new beatgrid format capable of handling not only beats, but also downbeats, phrases and sections as well varying tempos and time signatures have already opened a Pull Request each on our main repository. [#2844](https://github.com/mixxxdj/mixxx/pull/2844), [#2847](https://github.com/mixxxdj/mixxx/pull/)
+
+Cristiano which is focusing his work on the auto detection of such features is making weekly vlogs on youtube to share his progress with the community. The first one is available here:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/rVJ7pBWHp8E" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (Holzhaus) [inline pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html]:
```suggestion
{% include "video.html" with video_id="rVJ7pBWHp8E" %}
```

**Review by Holzhaus — CHANGES_REQUESTED** [review]

**Comment by crisclacerda** [comment]
Used new video tag, but apparently video has no sound, and controls does not appear.

File: pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+After one month of community bond period, last week they started coding. Cristiano and Harshit, who are both working towards the same final feature, a new beatgrid format capable of handling not only beats, but also downbeats, phrases and sections as well varying tempos and time signatures have already opened a Pull Request each on our main repository. [#2844](https://github.com/mixxxdj/mixxx/pull/2844), [#2847](https://github.com/mixxxdj/mixxx/pull/)
+
+Cristiano which is focusing his work on the auto detection of such features is making weekly vlogs on youtube to share his progress with the community. The first one is available here:
+
+{% include "video.html" with video_id="rVJ7pBWHp8E" %}
+
+Janek, who is working on a feature similar to serato flip, had the great idea and prepered a community call, where the new and old members alike had the opportunity to get to know each other better and put some faces to their usernames. 
+
+We are looking forward for a new call to discuss our GSoC projects with our students, mentors and the rest of the community.
+Stay tuned on [Zulip](https://mixxx.zulipchat.com/)
```
Comment (Holzhaus) [inline pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html]:
```suggestion
We are looking forward to a new call in which we discuss our GSoC projects with our students, mentors and the rest of the community.
If you're interested, hit us up on [Zulip](https://mixxx.zulipchat.com/).
```

File: pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+This year Mixxx is again taking part on Google Summer of Code, and have 3 students working full-time during 3 months to make Mixxx even more awesome.
+
+After one month of community bond period, last week they started coding. Cristiano and Harshit, who are both working towards the same final feature, a new beatgrid format capable of handling not only beats, but also downbeats, phrases and sections as well varying tempos and time signatures have already opened a Pull Request each on our main repository. [#2844](https://github.com/mixxxdj/mixxx/pull/2844), [#2847](https://github.com/mixxxdj/mixxx/pull/)
+
+Cristiano which is focusing his work on the auto detection of such features is making weekly vlogs on youtube to share his progress with the community. The first one is available here:
+
+{% include "video.html" with video_id="rVJ7pBWHp8E" %}
+
+Janek, who is working on a feature similar to serato flip, had the great idea and prepered a community call, where the new and old members alike had the opportunity to get to know each other better and put some faces to their usernames.
```
Comment (Holzhaus) [inline pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html]:
```suggestion
Janek, who is working on [a feature similar to Serato Flip](https://www.mixxx.org/wiki/doku.php/mixxx_macros), had the great idea and prepared a community call where the new and old members alike had the opportunity to get to know each other better and put some faces to their usernames. 
```

File: pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+{% block post %}
+
+{% load markup %}
+{% filter markdown:"extra" %}
+
+This year Mixxx is again taking part on Google Summer of Code, and have 3 students working full-time during 3 months to make Mixxx even more awesome.
+
+After one month of community bond period, last week they started coding. Cristiano and Harshit, who are both working towards the same final feature, a new beatgrid format capable of handling not only beats, but also downbeats, phrases and sections as well varying tempos and time signatures have already opened a Pull Request each on our main repository. [#2844](https://github.com/mixxxdj/mixxx/pull/2844), [#2847](https://github.com/mixxxdj/mixxx/pull/)
+
+Cristiano which is focusing his work on the auto detection of such features is making weekly vlogs on youtube to share his progress with the community. The first one is available here:
```
Comment (Holzhaus) [inline pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html]:
```suggestion
Cristiano which is focusing his work on the auto detection of such features is making weekly vlogs on YouTube to share his progress with the community. The first one is available here:
```

File: pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+date: 2020-06-08 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown:"extra" %}
+
+This year Mixxx is again taking part on Google Summer of Code, and have 3 students working full-time during 3 months to make Mixxx even more awesome.
```
Comment (Holzhaus) [inline pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html]:
```suggestion
This year Mixxx is again [taking part in Google Summer of Code]({% url "/news/2020-05-05-mixxx-gsoc-projects-2020.html" %}), and 3 students are working full-time for 3 months to make Mixxx even more awesome.
```

File: pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+After one month of community bond period, last week they started coding. Cristiano and Harshit, who are both working towards the same final feature, a new beatgrid format capable of handling not only beats, but also downbeats, phrases and sections as well varying tempos and time signatures have already opened a Pull Request each on our main repository. [#2844](https://github.com/mixxxdj/mixxx/pull/2844), [#2847](https://github.com/mixxxdj/mixxx/pull/)
```
Comment (Holzhaus) [inline pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html]:
```suggestion
After one month of community bonding period, they started coding last week. Cristiano and Harshit, who are both working towards the same final feature—new beatgrid format capable of handling not only beats, but also downbeats, phrases and sections as well varying tempos and time signatures—have already sent some inital patches to Mixxx' Git repository ([#2844](https://github.com/mixxxdj/mixxx/pull/2844), [#2847](https://github.com/mixxxdj/mixxx/pull/2847)).
```

**Review by Holzhaus — CHANGES_REQUESTED** [review]
Thank you for writing this! If you want you can also add some author metadata for yourself here: https://github.com/mixxxdj/website/blob/website/plugins/news.py#L13-L37

**Comment by Holzhaus** [comment]
> Used new video tag, but apparently video has no sound, and controls does not appear.

Should be fixed in #103.

**Comment by crisclacerda** [comment]
> Thank you for writing this! If you want you can also add some author metadata for yourself here: https://github.com/mixxxdj/website/blob/website/plugins/news.py#L13-L37

My pleasure, it's nice to contribute to the website as well. Thanks for the opportunity and for the video tag! 
Next time will be more careful with orthography, barely write English without a text speller, and don't have it on a text editor.
Should be all good now

File: pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown:"extra" %}
+
+This year Mixxx is again [taking part in Google Summer of Code]({% url "/news/2020-05-05-mixxx-gsoc-projects-2020.html" %}), and 3 students are working full-time for 3 months to make Mixxx even more awesome.
+
+After one month of community bonding period, they started coding last week. Cristiano and Harshit, who are both working towards the same final feature—new beatgrid format capable of handling not only beats, but also downbeats, phrases and sections as well varying tempos and time signatures—have already sent some inital patches to Mixxx' Git repository ([#2844](https://github.com/mixxxdj/mixxx/pull/2844), [#2847](https://github.com/mixxxdj/mixxx/pull/2847)).
```
Comment (Holzhaus) [inline pages/news/2020-06-08-mixxx-GSoC-Coding-Begins.html]: Sorry, looks like I missed a word :smile: 

```suggestion
After one month of community bonding period, they started coding last week. Cristiano and Harshit, who are both working towards the same final feature—a new beatgrid format capable of handling not only beats, but also downbeats, phrases and sections as well varying tempos and time signatures—have already sent some inital patches to Mixxx' Git repository ([#2844](https://github.com/mixxxdj/mixxx/pull/2844), [#2847](https://github.com/mixxxdj/mixxx/pull/2847)).
```

**Review by Holzhaus — CHANGES_REQUESTED** [review]

**Comment by Holzhaus** [comment]
Thank you!

**Pull request merged**